data/src/main/java/com/microsoft/azure/kusto/data/ClientRequestProperties.java

```diff
@@ -288,6 +285,25 @@ public static ClientRequestProperties fromString(String json) throws JsonProcess
         return null;
     }
 
+    private static Object unwrapJsonValue(JsonNode jsonNode) {
```

Comment:
is this for returning null ? what changed in behavior here?

Reply:
So before we had a subtle bug, when serializing the props and deserializing.

If we had a number there, it would read it as a string (because of line 274), and when it will try to parse it it will fail (since it's not in the proper timespan format). This makes sure that when deserializing it preserves the type of the object.